pages/11ty/sort.filter.js

```diff
@@ -8,6 +8,9 @@ module.exports = (config) => {
     return [...values].sort(comparer);
   };
 
+  /** filter by ignore */
+  const filterIgnore = (collection) => collection.filter( item => !item.data.ignore )
```

Comment:
it doesn't look to be a sort 11ty filter
it's better to create a separate module